Cargo.toml

```diff
@@ -27,7 +27,7 @@ input_buffer = { version = "0.4.0", git = "https://github.com/snapview/input_buf
 log = "0.4.8"
 rand = "0.8.0"
 sha-1 = "0.9"
-url = "2.1.0"
+url = "2.2.0"
```

Comment:
Does the changes in this PR _require_ (to build properly, or to work proberly) to use at lest url 2.2.0?

If not, I'd suggest not bumping this, cargo should happily pull url-2.2.0 even if we specify 2.1.0 (that's different to 0.x.y crates, where even minor version is considered incompatible).

Reply:
Apologies, this was a result of my misunderstanding of how Cargo decides which dependency version to fetch - I've changed it back now!